test/edu/wright/cs/raiderplanner/model/ApacheDerbyDatabaseTest.java

```diff
@@ -0,0 +1,158 @@
+/*
+ * Copyright (C) 2017 - Nathaniel Crossman
```

Comment:
Year should be 2018.

Reply:
Not sure why Eclipse generated the wrong year.